src/web/features.rs

```diff
@@ -206,8 +209,8 @@ mod tests {
 
     #[test]
     fn test_order_features_and_get_len_single_default() {
-        let default = Feature::new(DEFAULT_NAME.into(), Vec::new());
-        let non_default = Feature::new("non-default".into(), Vec::new());
+        let default = Feature::new(DEFAULT_NAME.into(), Vec::new(), Some(false));
+        let non_default = Feature::new("non-default".into(), Vec::new(), Some(false));
 
```

Comment:
Can you add some tests for the new behavior?

Reply:
There's no changes to the frontend, so no need for tests, I was confused. Testing this requires build tests (#822).